hasher-matcher-actioner/webapp/package.json

```diff
@@ -19,7 +19,9 @@
     "react-bootstrap": "^1.5.2",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3"
+    "react-scripts": "4.0.3",
+    "uplot": "^1.6.9",
+    "uplot-react": "^1.0.2"
```

Comment:
Installing this requires you to add `--legacy-peer-deps` argument to react because the authors have not explicitly allowed use with reacted 0.17. 

React + npm versions are a mess now. Signs of improvement, but all of this will take a while.